ringvax/app.py

```diff
@@ -169,7 +169,9 @@ def app():
     }
 
     sims = []
-    with st.spinner("Running simulation..."):
+    with st.spinner(
```

Comment:
This would be *really* cool as a progress bar; I can open a PR for that

Reply:
See #43 

It's nice when you have >1000 sims, but I agree that's probably overkill

Reply:
Or when you dial up `infection_rate` to max (useful for testing the warning/erroring)